packages/desktop/views/dashboard/governance/views/ProposalDetailsView.svelte

```diff
@@ -106,10 +106,14 @@
 
     async function setVotingEventPayload(eventId: string): Promise<void> {
         const event = await getVotingEvent(eventId)
-        if (event?.data?.payload?.type === ParticipationEventType.Voting) {
-            votingPayload = event.data.payload
+        if (event) {
+            if (event?.data?.payload?.type === ParticipationEventType.Voting) {
```

Comment:
we know that `event` exists, so we can remove the `?` 
```suggestion
            if (event.data?.payload?.type === ParticipationEventType.Voting) {
```